backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Helpers/PolicyHelper.cs

```diff
@@ -202,7 +202,7 @@ public static string GetAltinnAppDelegationPolicyPath(string org, string app, st
             }
             else
             {
-                return $"resourceregistry /{resourceRegistryId}/{offeredBy}/{coveredBy}/ delegationpolicy.xacml";
+                return $"resourceregistry/{resourceRegistryId}/{offeredBy}/{coveredBy}/resourcepolicy.xml";
```

Comment:
The delegation policy should still be called delegationpolicy.xml even if it's a delegation of a resource from the registry.
Otherwise the change in PolicyRepository.CreateBlobClient which use the policy name to determine which storage account to use will fail.